firedrake/tsfc_interface.py

```diff
@@ -103,6 +106,17 @@ def split(self, form):
     def multi_index(self, o):
         return o
 
+    def indexed(self, o, expr, multiindex):
```

Comment:
I just ran some tests without this bit. I think it's no longer useful. I'll remove it in my next commit.